examples/Python/PET/generate_noisy_data.py

```diff
@@ -0,0 +1,109 @@
+'''Poisson noise generation demo.
+
+Usage:
+  generate_noisy_data [--help | options]
+
+Options:
+  -f <file>, --file=<file>     prompts file [default: my_forward_projection.hs]
+  -p <path>, --path=<path>     path to data files, defaults to data/examples/PET
+                               subfolder of SIRF root folder
+  -o <file>, --output=<file>   output file for Poisson noisy data
+  -r <seed>, --seed=<seed>     random generator seed [default: 1]
+  -F <sf>, --sf=<sf>           scaling factor [default: 1.0]
```

Comment:
add something like "use a factor larger than 1 for higher relative noise" (or lower, I didn't check)

Reply:
done, though not sure I got it right - can you please run `examples/Python/PET/generate_noisy_data.py` to check?